device.js

```diff
@@ -24,6 +24,49 @@ class LoupedeckDevice extends EventEmitter {
 
         return rawDevices.flat()
     }
+    static thisDevice = "center";
+    setDevice(x) {
+        this.thisDevice = x;
```

Comment:
Work around. Ideally this would detect the device automatically

Reply:
Not sure I understand what this is for?